sample/TKSetupSamples.m

```diff
@@ -216,4 +216,65 @@ -(void)testProvisionDevice {
         return (member != nil);
     }];
 }
+
+-(void)testRecoverMember {
+    NSString *memberAliasString = self.payerAlias.value;
+
+    // Create a new SDK client with its own keystore to make sure
+    // we don't interfere with/use keys used to create the member
+    TokenIOBuilder *builder = [self sdkBuilder];
+    builder.keyStore = [[TKInMemoryKeyStore alloc] init];
+    TokenIO *tokenIO = [builder buildAsync];
+    __block int prompting = false;
+
+    void (^showPrompt)(NSString *s) = ^(NSString *s) {
+        prompting = true;
+    };
+
+    // beginRecovery begin snippet to include in docs
+    [tokenIO beginMemberRecovery:memberAliasString
+                       onSuccess:^() {
+                           // prompt user to enter code:
+                           showPrompt(@"Enter code emailed to you:");
+                       } onError:^(NSError *e) {
+                           @throw [NSException exceptionWithName:@"BeginRecoveryFailedException"
+                                                          reason:[e localizedFailureReason]
+                                                        userInfo:[e userInfo]];
+                       }];
+    // beginRecovery done snippet to include in docs
+
+    [self runUntilTrue:^ {
+        return (prompting != false);
+    }];
+
+    NSString *userEnteredCode = @"1thru6"; // unverified test user, so any code works
```

Comment:
Im thinking if "unverified" is not accurate. We do verify the test user, we just "skip the verification".
How about "the test users can bypass the verification, so any code works"

Reply:
Nice. Done.